### ISSUE TITLE
Invisible Ad takes over page. (All roads lead to 2017 Coke Sponsored LoL Finals)

### ISSUE THREAD

**Issue by ApproachingApathy** [Nexus-Mods/web-issues#203]
URL: https://rd.nexusmods.com/skyrimspecialedition/mods/categories/# (Throughout the site)
Bug: Clicking anywhere in a page leads to this [Webpage](http://www.cokeworlds2017.com/?campaign=Curse)
Reproduction: Has happened on many pages, follows into incognito tab. Using Chrome.
Expected: To click on a inter-site link. e.g. to go to category page.
Result: Goes to this: [http://www.cokeworlds2017.com/?campaign=Curse](http://www.cokeworlds2017.com/?campaign=Curse)

**Comment by rscottexe** [comment]
Hello, sorry about this issue.

What country are you based in? The ad will be geotargeted to your country and I can't see it here in the UK. If you let me know I can use a VPN to test it and get it sorted ASAP.

**Comment by Trizaratops** [comment]
I've had the exact same problem, and experienced both ads. I couldn't even click the link to return to the old version because it was also an advertisement. I am also in the US, and I am using Chrome.

**Comment by verenion** [comment]
This issue should have been sorted.

**Comment by Trizaratops** [comment]
Nope. Even trying to switch to the new layout sends me to the Coke LoL page.

**Comment by rscottexe** [comment]
Trizaratops, please try doing a hard refresh of the site (CTRL+F5) a few times and let me know if this issue persists.

**Comment by Trizaratops** [comment]
Yes, it persists. And specifically I'm encountering the problem on this
mod's page:

https://www.nexusmods.com/skyrimspecialedition/mods/13152/?

**Comment by rscottexe** [comment]
Ok, so that's on the current site and not on the new one.

Is this ONLY happening on that "View new layout" link at the top, or is it happening anywhere you click on the page?

**Comment by Trizaratops** [comment]
It was only the View New Layout button. I was able to download the mod yesterday with no issues on the old version.

That said, I just checked again at 11:57 CST and it’s fine. I’m able to navigate to the new version without any advertisement interference. That said, clicking “view new layout” on any page redirects not to that page in the new version, or even that game in the new version, but to the main page.
